analytics/tests/test_views.py

```diff
@@ -21,6 +21,7 @@
     MultiuseInvite,
     PreregistrationUser,
     Realm,
+    RealmAuditLog,
```

Comment:
Don't add tests separately. It should always go along with the associated commit.